index.bs

```diff
@@ -16,13 +16,439 @@ WPT Display: closed
+
+## Deeper CORS Integration ## {#deeper-cors-integration}
+
+It is tempting to design this specification such that it piggy-backs and/or integrates with CORS deeply, since CORS intuitively feels like it is meant to address a similar problem of enabling cross-origin functionality. However, this would be undesirable for a few reasons:
+
+* If CORS (and the relevant SAA permission, of course) were a "sufficient" condition for attaching unpartitioned cookies...
+    * Then this would allow the top-level site to attack the embedded site by sending (CORS-enabled) credentialed requests to arbitrary endpoints on the embedded site, without requiring any opt-in from the embedded site before it received those requests. This would make CSRF attacks against the embedded site more feasible. This is undesirable for security reasons.
+* If CORS were required for the user agent to attach unpartitioned cookies to the request...
+    * Then this would mean the embedded site would be required to allow the top-level site to read the bytes of its responses and response headers, just so that the user agent would include cookies when fetching the embedded resource. This is a more powerful capability than simply attaching unpartitioned cookies, so this would expose the embedded site to unnecessary attack vectors from the top-level site. This is undesirable for security reasons.
+    * This would also mean that in order to fix an embedded widget on some page, the top-level site must perform some action to enable CORS; the embedded site alone would be unable to update the page and fix the widget. This is undesirable from a developer usability / composability standpoint.
+
+Therefore, CORS ought to be neither necessary nor sufficient for attaching unpartitioned cookies to a cross-site request. This specification is therefore designed to be orthogonal to CORS.
```

Comment:
Maybe briefly mention that it shares relying on the Origin header as a signal for developer, but that we consider this a positive thing since we're able to reuse a header that sends the same information, in both cases for security purposes?

Reply:
Done